### DISCUSSION TITLE
Change positions of brackets in dump()

### DISCUSSION THREAD

**Discussion by supsm** [nlohmann/json#3022] [Ideas]
There should be an option to change the location of where the brackets go in the `dump` function.  
  
Currently, all jsons look something like this:
```json
"object": {
    "key": "value"
}
```
It would be cool if we could have jsons also look like this:
```json
"object":
{
    "key": "value"
}
```

**Comment by nlohmann** [comment]
This goes to the long list of detailed feature requests for the dump function. The problem is to find an API that allows to specify call these aspects, yet still remains extendable such that we do not have breaking changes down the road.
